contao/languages/en/tl_metamodel_dcasetting_condition.php

```diff
@@ -21,13 +21,13 @@
 /**
  * Legends
  */
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['title_legend']  = 'Title';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['config_legend'] = 'Config';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['basic_legend']  = 'Basic config';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['config_legend'] = 'Condition config';
```

Comment:
Maybe we want to put the whole word `configuration` here instead of an abbreviation.